engine/loader.asm

```diff
@@ -889,11 +901,27 @@ SpriteLoop@
             std         ObjCodePtr@
             addd        GroupObjCodeSize@
  IFDEF DEBUG
-            cmpd        #$8000
-            bls         >
-            swi                                 * Error: out of memory for level / object code
+            cmpd        #$8000-OBJPAGEGUARD     * are we beyond the page - the guard space?
```

Comment:
I think you need to remove the "IFDEF DEBUG" macros around this block, or else this will not work properly in Release mode.

Reply:
TY this is a bug. I am moving the IFDEF to the code block just below so that the out of memory test is preserved.

Reply:
Fixed